lib/mina/application.rb

```diff
@@ -27,8 +29,9 @@ def sort_options(options)
 
     def top_level_tasks
       return @top_level_tasks if @top_level_tasks.include?('init')
-      @top_level_tasks << :debug_configuration_variables
-      @top_level_tasks << :run_commands
+
+      @top_level_tasks << 'debug_configuration_variables'
```

Comment:
All `@top_level_tasks` are strings so I modified the default task names. It worked before because Rake calls `to_s` on a task name ([ref](https://github.com/ruby/rake/blob/a842fb2c30cc3ca80803fba903006b1324a62e9a/lib/rake/application.rb#L164))